overlord/hookstate/ctlcmd/get.go

```diff
@@ -57,36 +66,42 @@ Nested values may be retrieved via a dotted path:
 
     $ snapctl get author.name
     frank
+
+Values of plug or slot attributes may be printed in plug hooks via:
```

Comment:
```
Values of interface connection settings may be printed with:

    $ snapctl get :myplug usb-vendor
    $ snapctl get :myslot path

This will return the named setting from the local interface endpoint, whether a plug
or a slot. Returning the setting from the connected snap's endpoint is also possible
by explicitly requesting that via the --plug and --slot command line options:

    $ snapctl get :myplug --slot usb-vendor

This requests the "usb-vendor" setting from the slot that is connected to "myplug".
```